db/views/reporting_overdue_patients_v01.sql

```diff
@@ -0,0 +1,89 @@
+SELECT DISTINCT ON (reporting_patient_states.month_date, reporting_patient_states.patient_id)
+  reporting_patient_states.month_date AS month_date,
+  reporting_patient_states.patient_id AS patient_id,
+  reporting_patient_states.assigned_facility_id AS assigned_facility_id,
```

Comment:
We should add the other region ids and slugs for block, district and state so that filtering a querying on the table is easier in metabase and other ad-hoc reports